src/goto-programs/interpreter_evaluate.cpp

```diff
@@ -299,6 +301,9 @@ bool interpretert::memory_offset_to_byte_offset(
   }
 }
 
+/// Evaluate an expression
```

Comment:
I'm not sure this summary adds much, consider elaborating on what evaluating an expression means